shared/bazel/rules/cc_rules.bzl

```diff
@@ -400,3 +600,86 @@ def wpilib_cc_static_library(
         srcs = ["//:license_pkg_files", name + "-static.pkg"],
         tags = ["no-remote", "manual"],
     )
+
+def _generate_def_windows_impl(ctx):
+    """
+    This is a modified version of built in cc_static_library implementation
+    https://github.com/bazelbuild/bazel/blob/8.2.1/src/main/starlark/builtins_bzl/common/cc/experimental_cc_static_library.bzl
+
+    The built in version amalgamates all of the transative dependency objects into a single shared library. However, we do not want our
```

Comment:
Something about this comment scans wrong to me; is there a missing or extra "not" somewhere in there? There may also be subtleties about windows Def files that mean that I'm just not understanding correctly; if that is the case, some elaboration may be helpful.

Reply:
Worse than that, this is a copy from another function.  Rewritten in the def file review.